src/app/devices/deviceContent/components/deviceSettings/deviceSettingsPerInterface.tsx

```diff
@@ -62,22 +64,27 @@ export default class DeviceSettingsPerInterface
         return (
             <LocalizationContextConsumer>
                 {(context: LocalizationContextInterface) => (
-                    <div className="pnp-detail-list">
-                        <div className="list-header">
-                            <span className="column-name">{context.t(ResourceKeys.deviceSettings.columns.name)}</span>
-                            <span className="column-schema-sm">{context.t(ResourceKeys.deviceSettings.columns.schema)}</span>
-                            <span className="column-unit">{context.t(ResourceKeys.deviceSettings.columns.unit)}</span>
-                            <span className="column-value">{context.t(ResourceKeys.deviceSettings.columns.reportedValue)}</span>
-                            <DefaultButton
-                                className="column-toggle"
-                                onClick={this.onToggleCollapseAll}
-                            >
-                                {this.state.allCollapsed ?
-                                    context.t(ResourceKeys.deviceSettings.command.expandAll) :
-                                    context.t(ResourceKeys.deviceSettings.command.collapseAll)}
-                            </DefaultButton>
+                    <div className="pnp-detail-list scrollable-lg ms-Grid">
+                        <div className="list-header ms-Grid-row">
+                            <span className="ms-Grid-col ms-u-sm3">{context.t(ResourceKeys.deviceSettings.columns.name)}</span>
+                            <span className="ms-Grid-col ms-u-sm2">{context.t(ResourceKeys.deviceSettings.columns.schema)}</span>
+                            <span className="ms-Grid-col ms-u-sm2">{context.t(ResourceKeys.deviceSettings.columns.unit)}</span>
+                            <span className="ms-Grid-col ms-u-sm4 reported-value">
+                                {context.t(ResourceKeys.deviceSettings.columns.reportedValue)}
+                                <TooltipHost
```

Comment:
1. adding tooltip to indicate user need to refresh
2. Collapse all/expand all button changes to a icon button to fix horizonal scrollbar

![image](https://user-images.githubusercontent.com/5489222/64989523-8e85eb00-d882-11e9-843c-370b46ada3c5.png)